src/FileHandler.php

```diff
@@ -156,20 +158,20 @@ public function cleanUp($zipFile)
     public function renameFiles($manifest = null)
     {
         $bindings = [
-            [':uc:vendor', ':uc:package', ':lc:vendor', ':lc:package'],
```

Comment:
I like the more generic way of writing more than using the MyVendor etc. examples.

Reply:
The way this was done before, these were acting on strings in an array in `rewriteRules.php`. However with the iterators they instead act on the filenames themselves. 

Since dealing with special characters can be a bit of a pain in terminal, it seemed easier use "MyPackage" for the replacements and save the step of creating a redundant file that merely linked your existing `/src/MyPackageServiceProvider.php` through rewriteRules.php's line `'src/MyPackageServiceProvider.php' => 'src/:uc:packageServiceProvider.php',`.


What I would actually like to see—after realizing Wrapping.php is mostly just filling—, is combining this iterator with the other iterator and running fill and rename in one pass.